docs/contributing/github_integration.rst

```diff
@@ -1,47 +1,35 @@
+GitHub Integrations
 ====================
 
-buildtest has several github integration, including automated checks during PR that maintainers will check to
+buildtest has several github integration, including automated checks during PR that maintainers will check
 during the PR review. You should check results from the `buildtest actions <https://github.com/HPC-buildtest/buildtest-framework/actions>`_
-to see if any fixes need to me made.
+that are also typically linked as part of the pull request testing suite.
 
 It's good practice to check Travis `builds <https://travis-ci.com/HPC-buildtest/buildtest-framework>`_ since we use Travis
 to run regression test and Codecov and Coveralls depend on Travis to pass all checks.
 
 You will want to make sure code is formatted via black as we have automated checks for python formatting. If you have not
 setup the black hook check out :ref:`black_hook`
 
+If you notice the black linter step in `GitHub Actions <https://github.com/HPC-buildtest/buildtest-framework/actions>`_ is
+failing, make sure you have the right version of black installation.
+
 GitHub Apps
 ------------
 
 The following apps are configured with buildtest.
 
-- **Travis CI** - Test and deploy with confidence. Trusted by over 800,000 users, Travis CI is the leading hosted continuous integration system.
-
-- **CodeCov** - Codecov provides highly integrated tools to group, merge, archive and compare coverage reports
-
-- **Coverall** - Coveralls is a web service to help you track your code coverage over time, and ensure that all your new code is fully covered.
-
-- **CodeFactor** - CodeFactor instantly performs Code Review with every GitHub Commit or PR. Zero setup time. Get actionable feedback within seconds. Customize rules, get refactoring tips and ignore irrelevant issues.
-
-- **GuardRails** - GuardRails provides continuous security feedback for modern development teams
-
-- **Snyk** - Snyk tracks vulnerabilities in over 800,000 open source packages, and helps protect over 25,000 applications.
-
-Links to the following apps
-
-- **Travis**: https://travis-ci.com/HPC-buildtest/buildtest-framework
-
-- **CodeCov**: https://codecov.io/gh/HPC-buildtest/buildtest-framework
+- `Travis CI <https://travis-ci.com/HPC-buildtest/buildtest-framework>`_ - Test and deploy with confidence. Trusted by over 800,000 users, Travis CI is the leading hosted continuous integration system.
 
-- **Coveralls**: https://coveralls.io/github/HPC-buildtest/buildtest-framework
+- `CodeCov <https://codecov.io/gh/HPC-buildtest/buildtest-framework>`_ - Codecov provides highly integrated tools to group, merge, archive and compare coverage reports
 
-- **CodeFactor**: https://www.codefactor.io/repository/github/hpc-buildtest/buildtest-framework
+- `Coveralls <https://coveralls.io/github/HPC-buildtest/buildtest-framework>`_ - Coveralls is a web service to help you track your code coverage over time, and ensure that all your new code is fully covered.
 
```

Comment:
This is much cleaner! One question that comes to mind is that (for some future update) if we can store the repository address / name as a variable, so we don't need to do manual changes for links like this. I'm not sure the extent to which this is possible with sphinx, but maybe? Either way, that's out of scope for here, this looks good.